Source/Cogworks.Essentials/Cogworks.Essentials.csproj

```diff
@@ -6,7 +6,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="5.0.0" />
-    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.8.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.10.2" />
+    <PackageReference Include="System.Runtime.Caching" Version="5.0.0" />
```

Comment:
Hmm thinking if we need it as we have `Microsoft.Extensions.Caching.Memory` or if we should add `https://www.nuget.org/packages/Microsoft.Extensions.Caching.Abstractions/` 🤔 

Another thing is if we should add those dependencies to nuspec ? 🤔 cc: @mzajkowski